js/videoBlock.js

```diff
@@ -0,0 +1,58 @@
+const videos = [
+    {
+        title: "Miss Bee, a teen singer with a beautiful voice: High hopes",
```

Comment:
👍 looking good. I do think we'll be able to get title and video id dynamically eventually so this is great data to start with.